docker-compose.jsonnet

```diff
@@ -358,6 +359,13 @@
     state_path:: "./prizebot_state.json",
   },
 
+  youtubebot:: {
+    zulip_email: "youtube-bot@chat.videostrike.team",
+    zulip_api_key: "",
+    google_credentials_file: $.cutter_creds_file,
+    channel_id: "UCz5-PNQxaT4WtB_OMAwD85g", // DesertBusForHope
```

Comment:
I assume this is something publicly available.

Reply:
yes, it's the channel id of the DB channel. anyone can look it up via the api based on the public channel name.